terraform/files/byoh.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sudo apt-get install -y socat ebtables ethtool conntrack
+
+# To do: do we need to add host name to /etc/hosts
+cat /etc/hosts
+
+curl -L https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/releases/download/v0.3.0/byoh-hostagent-linux-amd64 -o ~/byoh-hostagent-linux-amd64
+chmod +x ~/byoh-hostagent-linux-amd64
+
+curl -fsSL https://tailscale.com/install.sh | sh
+
+sudo tailscale up --accept-routes --authkey "$AUTH_KEY"
```

Comment:
we could probably remove this here and just have the tailscale.sh added to the byoh remote exec in main.tf

but it is not a huge amount of repetition so not really bothered

Reply:
Agreed  that we could avoid repetition.

Reply:
i've got a lot of work to do to consolidate all the terraforms, so any on-off improvements can be skipped for now